runhouse/servers/obj_store.py

```diff
@@ -1514,28 +1521,193 @@ def status_local(self):
                     else (py_module + "." + cls.__qualname__)
                 )
                 if isinstance(v, runhouse.Resource):
-                    resources_in_env_modified.append(
+                    objects_in_env_modified.append(
                         {"name": k, "resource_type": cls_name}
                     )
                 else:
-                    resources_in_env_modified.append(
+                    objects_in_env_modified.append(
                         {"name": k, "resource_type": cls_name}
                     )
-            return resources_in_env_modified
-        else:
-            return []
+
+        if env_name:
+            list_env_actor = (
```

Comment:
naming confused me initially, this list will either be empty or have one entry?

Reply:
yeah, cause we filter the ray list actors CLI by `f"name={env_name}"`. Renamed to env_actor.